subjects/05-props-v-state/exercise.js

```diff
@@ -75,13 +71,62 @@ const Tabs = React.createClass({
 
 })
 
+const StatefulTabs = React.createClass({
```

Comment:
:+1: 

It feels like we could have saved some time/discussions by using this pattern.